RecoEgamma/EgammaIsolationAlgos/src/EgammaRecHitIsolation.cc

```diff
@@ -91,7 +91,7 @@ double EgammaRecHitIsolation::getSum_(const reco::Candidate* emObject,bool retur
       for (CaloSubdetectorGeometry::DetIdSet::const_iterator  i = chosen.begin ();i != chosen.end (); ++i){ //loop selected cells
 	j = caloHits_.find(*i); // find selected cell among rechits
 	if(j != caloHits_.end()) { // add rechit only if available 
-	  auto const cell  = theCaloGeom_.product()->getGeometry(*i);
+	  auto cell  = (theCaloGeom_.product())->getGeometry(*i);
```

Comment:
here and below the expression can be simplified to ```theCaloGeom_->getGeometry(*i);``` (since it's touched anyways)